internal/command/tokens/create.go

```diff
@@ -59,45 +61,82 @@ func newDeploy() *cobra.Command {
 			Description: "The duration that the token will be valid",
 			Default:     time.Hour * 24 * 365 * 20,
 		},
+		flag.String{
+			Name:        "scope",
+			Shorthand:   "s",
+			Description: "either 'app' or 'org'",
+			Default:     "app",
+		},
```

Comment:
I don't have strong feelings about flyctl UX, but are we sure we want `fly tokens create deploy -s org` and not `fly tokens create org`?